mon-pix/app/components/authentication/password-reset-demand-form.gjs

```diff
@@ -0,0 +1,122 @@
+import PixButton from '@1024pix/pix-ui/components/pix-button';
+import PixButtonLink from '@1024pix/pix-ui/components/pix-button-link';
+import PixInput from '@1024pix/pix-ui/components/pix-input';
+import PixMessage from '@1024pix/pix-ui/components/pix-message';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { t } from 'ember-intl';
+import ENV from 'mon-pix/config/environment';
+
+import isEmailValid from '../../utils/email-validator.js';
+
+export default class PasswordResetDemandForm extends Component {
+  @service intl;
+
+  @tracked isLoading = false;
+  @tracked errorMessage;
+  @tracked emailInputPlaceholder = this.intl.t(
+    'components.authentication.password-reset-demand-form.fields.email.placeholder',
+  );
+  @tracked emailInputvalidationStatus;
+  @tracked emailInputvalidationErrorMessage;
+
+  email;
+
+  @action
+  handleEmailChange(event) {
+    this.email = event.target.value;
+    this.emailInputvalidationStatus = isEmailValid(this.email) ? 'success' : 'error';
+    this.emailInputvalidationErrorMessage = this.intl.t(
+      'components.authentication.password-reset-demand-form.fields.email.error-message-invalid',
+    );
+  }
+
+  @action
+  async handlePasswordResetDemand(event) {
+    if (event) event.preventDefault();
+
+    this.errorMessage = null;
+    this.isLoading = true;
+
+    const email = this.email.trim();
+    if (!email) {
+      return;
+    }
```

Comment:
En faisant des tests fonctionnels, j'ai réussi à soumettre le formulaire avec un email invalide (ex: `asdsad@asdsad`). Ce qui fait apparaître le message suivant: "Une erreur interne est survenue, nos équipes sont en train de résoudre le problème. Veuillez réessayer ultérieurement."

<img width="724" alt="image" src="https://github.com/user-attachments/assets/e63d7788-251a-44e2-8530-f0ea75195d0a">

Je pense qu'il faudrait vérifier le status de validation de l'email au moment de la soumission du formulaire, et l'empêcher si la validation est en erreur:
```suggestion
    const email = this.email.trim();
    if (!email || this.emailInputvalidationStatus === 'error') {
      return;
    }
```

Reply:
C'est corrigé !

Reply:
C'est ok, le formulaire n'est plus soumis mais le loader du bouton reste actif et on ne peut plus cliquer sur le bouton.

<img width="746" alt="image" src="https://github.com/user-attachments/assets/75ec9ba1-24a8-410e-9305-a4c31f7d85a9">

Il faudrait mettre le `this.loading = true` après le `if` (une fois qu'il est passé)

Reply:
Je n'ai pas eu ce comportement avec mes tests sur Firefox et là je ne l'ai pas reproduis avec Chrome non plus. Ce qui fait que je n'y ai pas pensé. Ceci dit c'est clairement une chose à faire. Je suis désolé de l'avoir laissé passée !
C'est corrigé !